### ISSUE TITLE
Old Working Copy not erased

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#144]
```
What steps will reproduce the problem?
1. Create Working Copy
2. Delete it
3. Create same Working Copy

What is the expected output? What do you see instead?
Show the old working copy, should be the newest.


What version of the product are you using? On what operating system?
SVNX 1.3.1 on Mac OS X Lion

Please provide any additional information below.


```

Original issue reported on code.google.com by `fouquet....@gmail.com` on 23 Aug 2011 at 9:44

**Comment by GoogleCodeExporter** [comment]
```
I don’t understand what the problem is.

> 1. Create Working Copy
By which means do you create the working copy?
[Drag & drop within the list or from the Finder, automatically via a checkout, 
or ‘+’ button & manual entry?]

> Show the old working copy, should be the newest.
How do you “show the old working copy” if you just deleted it?
What do you mean by ”newest”?  Are you referring to repository revisions?
What difference do/should you see between the old & new working copies?
[Your description state that they are the same.]

When you use the term "Working Copy" do you mean an item in the "Working 
Copies" list window?
Or do you mean an svnX working copy window, or a working copy folder in the 
Finder?
```

Original comment by `chris...@gmail.com` on 23 Aug 2011 at 11:34

**Comment by GoogleCodeExporter** [comment]
```
I don’t understand what the problem is.

> 1. Create Working Copy
By which means do you create the working copy?
[Drag & drop within the list or from the Finder, automatically via a checkout, 
or ‘+’ button & manual entry?]
->Creation of the working copyautomatically via a checkout.

> Show the old working copy, should be the newest.
How do you “show the old working copy” if you just deleted it?
What do you mean by ”newest”?  Are you referring to repository revisions?
What difference do/should you see between the old & new working copies?
[Your description state that they are the same.]
-> After delete the working copy, I re-Create the same working copy 
automatically with checkout, is the same path (folder deleted by myself just 
before re-create working copy). Then I notice that hasn't checkout well, I mean 
files are there, but in Working Copy Windows, it shows me the same items (files 
to add, conflicts, etc.) I had before I deleted the working copy.
-> I solve the problem! I've deleted the "com.lachoseinteractive" files stored 
in "Caches" and "Preferences" folders and I restart SVNX, the re-create my 
Working Copy automatically via checkout in the same path worked well, and no 
more items as "conflicts" or "files to add" are present.

->That it helps you to understand what is the bug ?

```

Original comment by `fouquet....@gmail.com` on 24 Aug 2011 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
I don’t think this is a bug in svnX.

Each checkout creates a NEW working copy.  It’s possible (& quite common) to 
have multiple working copies of the same part of a repository.
Deleting a WC bookmark/item in svnX has no effect on the actual WC on disc.
Also, you must refresh an svnX WC window for it to reflect any changes to the 
WC status.

Each new checkout (by definition) creates files that are up to date.
If you are seeing differences between your checkout & the repo before you do 
anything then you are either looking at the wrong WC or you didn’t checkout 
correctly.
[In svnX 1.x you can checkout a folder by simply dragging it into a Finder 
window.
 If you use the Checkout toolbar button instead you MUST also create a new folder,
 otherwise the checkout is merged into the target folder.  This may be what you are seeing.]

The contents of ~/Library/Caches/com.lachoseinteractive.svnX caches data about 
the repo.
It has nothing to do with the working copies.  Each WC is contained in its 
entirety in its folder in the Finder.

In summary, I don’t know what you are expecting Subversion & svnX to do here,
but I’m pretty sure svnX is doing it correctly.
Can you give a specific set of steps that cause a specific problem?
Have you read the svnX help docs & Subversion book?


```

Original comment by `chris...@gmail.com` on 24 Aug 2011 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
I'll try to simplify the problem : I can't properly delete a WC bookmark.

I had conflicts on a WC (I don't remember how it appends).
I've deleted the WC bookmark (on SVNX).
I've deleted the WC folder (on Finder).
I've checked out the repo in the same path. I had to create a new folder (I use 
the same name as the one which was previously deleted).
I expected to have an clean WC bookmark (no conflicts, no files to add, no 
files modified) but instead it showed conflicts , files to add and files 
modified.

So I think, there is a file that store modifications of a WC, and the file is 
not update when an WC bookmark is deleted. Am I wrong by thinking this ?

I compared its use with TortoiseSVN on Windows : when I deleted a WC, and check 
out in the same path, I had not this problem.
```

Original comment by `fouquet....@gmail.com` on 24 Aug 2011 at 2:45

**Comment by GoogleCodeExporter** [comment]
```
This is not possible.  If you delete a WC folder - then it’s gone.

> So I think, there is a file that store modifications of a WC, and the file is 
not update when an WC bookmark is deleted. Am I wrong by thinking this ?
You are wrong.  (As I stated previously.)

If you delete a WC folder then ask svnX to open a bookmark to it you will get 
an error stating that it could not be opened.  If a WC window was open in svnX 
when you deleted its WC folder then it will simply display stale info.  When 
you refresh the window you will see a similar error.  You should close that 
window.

If you are seeing modified (etc.) files then you can click the Reveal toolbar 
item to show the file in the Finder & the Diff toolbar item to see the diffs.  
If svnX is indicating that there are mods, conflicts etc. it is because 
Subversion is telling it so.  Are you sure that the folder you deleted/replaced 
in the Finder is the same one you are viewing in svnX?  Use Reveal or ⌘-click 
the svnX WC window’s title & choose the second menu item.

Could you be experiencing a strange Lion Versions 
<http://www.apple.com/macosx/whats-new/auto-save.html> interaction?
```

Original comment by `chris...@gmail.com` on 24 Aug 2011 at 4:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `chris...@gmail.com` on 1 Jan 2012 at 7:12
- Changed state: **Invalid**
